components/navbar/navbar.snip.html

```diff
@@ -22,12 +22,17 @@
     <a role="button" on="tap:{{sidebar-id}}.toggle"class="ampstart-navbar-trigger {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} pr2">☰</a>
     {{#logo}}
       {{#href}}
-      <a href="{{.}}" class="text-decoration-none inline-block">
+      <a href="{{.}}" class="text-decoration-none inline-block mx-auto">
```

Comment:
what happens if there are items on the `<nav>` that follows this (and the nav is being displayed.)